Q/Qemu/build_tarballs.jl

```diff
@@ -57,38 +69,85 @@ platforms = [
 ]
 platforms = expand_cxxstring_abis(platforms)
 
+# some platforms need a newer glibc, because the default one is too old
+glibc_platforms = filter(platforms) do p
+    libc(p) == "glibc" && proc_family(p) in ["intel", "power"]
```

Comment:
Did you want to run this filter over all available platforms? Right now it's only x64 and i686 Linux anyways.

Reply:
Just to keep the code generic -- the same pattern exists in other recipes where glibc is upgraded (crun, systemd).